docs/start/getting-started/fragments/ios/build.md

```diff
@@ -0,0 +1,5 @@
+## What we'll build
```

Comment:
Looks like there is some new content being added in this PR, that isn't just replacing the `.callout`s, no?

Reply:
Yeah. I pushed another update last night and since this PR was not merged it updated the pull request.